pyproject.toml

```diff
@@ -62,9 +62,9 @@ dependencies = [
   "mmh3",  # fast hashing function (murmurhash3)
   "quantulum3",  # quantities extraction from text
   "posthog",  # telemetry
-  "azure-ai-formrecognizer==3.2.0b2",  # forms reader
+  "azure-ai-formrecognizer>=3.2.0b2",  # forms reader
   # audio's espnet-model-zoo requires huggingface-hub version <0.8 while we need >=0.5 to be able to use create_repo in FARMReader
-  "huggingface-hub<0.8.0,>=0.5.0",
```

Comment:
How did you test this, specifically? Did the audio nodes install and work properly once you installed without this pin?

Reply:
Audio tests are had been run manually. Also, going to the espnet-model-zoo, you will see that there is no need for lower hub versions anymore.